### PR TITLE
Rename VERSION => STABLE_VERSION for the unification at the changed tool

### PR DESCRIPTION
**Pull request by wilzbach**
Small-scale version of https://github.com/dlang/dlang.org/pull/1821, s.t. we can move step by step.

### CODE REVIEW

**Comment by wilzbach** [comment]
As mentioned before, if `change.d` would be part of this repo I wouldn't need to go through these hops to please DAutoTest)...

File: changelog/changelog.ddoc [hunk 1]
```diff
@@ -15,14 +15,15 @@ SUBNAV=$(SUBNAV_TEMPLATE
 SEARCH_OPTIONS_EXTRA=<option value="dlang.org/changelog" selected>Change
 Log</option>
 
-VERSION=
+STABLE_VERSION=
```
Comment (wilzbach) [inline changelog/changelog.ddoc]: To be fair: it wouldn't be too hard to continue to generate `VERSION` at the `changed.d` script, but imho this helps clarifying things.

**Comment by MartinNowak** [comment]
Let's wait a sec and properly think this through before churning more code.
https://github.com/dlang/dlang.org/pull/1821#issuecomment-315603750

File: changelog/changelog.ddoc (hunk 1 above)
Comment (MartinNowak) [inline changelog/changelog.ddoc]: Not sure I have the same understanding of STABLE_VERSION as you, so far we used that term to pin tools/compilers, but not for anything in our docs/targets.

**Pull request closed without merging**